install.sh

```diff
@@ -267,6 +317,11 @@ parse_command_line_arguments() {
             INTERNAL_INSTALL="true"
         elif [ "$p" = "--internal-postupdate" ]; then
             INTERNAL_POSTUPDATE="true"
+        elif [ "$p" = "--internal-updated-from" ]; then
+            : # nothing to do here
+        elif [ "$prev_p" = "--internal-updated-from" ] && (echo "$p" | grep -qvE '^\-\-.*'); then
+            # INTERNAL_UPDATED_FROM_COMMIT="$p" # not yet used !
```

Comment:
Why do we need this here if it's unused?

Reply:
Because: We have no chance of getting the commit SHA from the update when a user updated from an old version which doesnt hand over the SHA to the **updated** install.sh, and such we also dont have it when we introduce a change. So from now on we have this SHA hash, also we dont use it yet. Hopefully we never need to really use it =)

Reply:
Fair enough. I don't think it hurts to have it, though also unsure how we could use it in the future. 🙂

Reply:
Future: We need an SHA hash before update to check if we updated through critical commits to trigger legacy transform.